### PR TITLE
Express Setup with Mongo

### LINKED ISSUE #22: Backend Initialization

### PR DESCRIPTION
**Pull request by IsitaB**
Testing this PR

Run server in one terminal
`cd server`
`nodemon server`

Run frontend in another terminal
`npm start`

Be sure to get the right config file

Confirm functionality for the mongoose and port connection. No models/routes have been created in this PR

### CODE REVIEW

**Review by JoshuaLawson8 — APPROVED** [review]
It looks like there was just a lot deleted and a gitignore added. Init push

**Comment by JoshuaLawson8** [comment]
I'm not really sure what was done here

**Review by charlotte-zhuang — APPROVED** [review]
I think this is good? Does this just add things to the gitignore that were missed?

**Comment by thibui0908** [comment]
Add additional files to gitignore and we should be good

**Review by thibui0908 — APPROVED** [review]

**Comment by euniceyoh** [comment]
i can't really tell what changes in backend occurred? and i think node_modules should have been ignored, so maybe we should fix that before merging?

**Comment by IsitaB** [comment]
I need to rebase

this things from pre gitignore on main times

server/node_modules can be fixed. I guess it got cached

**Comment by IsitaB** [comment]
> 

I actually deleted the accidentally committed node_modules, not the other way around. we're chilling

File: .gitignore [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
+
 # testing
 /coverage
 
 # production
 /build
 
 # misc
-.DS_Store
+*/.DS_Store
```
Comment (charlotte-zhuang) [inline .gitignore]: undo change please, i think `.DS_Store` is correct

**Review by charlotte-zhuang — CHANGES_REQUESTED** [review]
Need to remove .DS_Store and see comment

**Review by christinang3030 — APPROVED** [review]
The changes made looks good to me.

File: .gitignore (hunk 1 above)
Comment (IsitaB) [reply .gitignore]: oh my ide was trying to push ds_store in some sub_dir. **/.DS_Store should gitignore all of them

**Pull request merged**